examples/lava/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "private": true,
+  "dependencies": {
+    "lava": "^0.13.2",
+    "director": "~1.2.8",
+    "todomvc-app-css": "=1.0.0",
```

Comment:
It's not a bug, it's intended.
This example does not use todomvc-app-css at all, so maybe it's better to remove it completely?

My framework is unable to work with IDs in CSS files, so I have rewritten your index.css into app.css: all ID selectors were renamed into class selectors.

For example: #todoapp -> .todoapp

And one exception:
label[for='toggle-all'] -> .toggle-all-label

P.S. Topic about the framework
https://github.com/tastejs/todomvc/issues/1084